src/pseudopeople/interface.py

```diff
@@ -162,7 +162,7 @@ def generate_decennial_census(
     source: Union[Path, str] = None,
     seed: int = 0,
     config: Union[Path, str, Dict[str, Dict]] = None,
-    year: int = 2020,
+    year: Union[int, None] = 2020,
```

Comment:
Thanks, this is what tripped me up. Also, note that you can use `Optional[int]` as well which is slightly simpler to type out.

Reply:
@rmudambi had the same comment, but I'm not sure they're equivalent in this case. https://github.com/ihmeuw/pseudopeople/pull/118#discussion_r1174138530

Reply:
Nevermind, this is good suggestion, updated.